linode/manager
# DBAAS1-85: Database Details Backups Tab

## Preview

<img width="1504" alt="Screen Shot 2021-12-22 at 12 17 24 PM" src="https://user-images.githubusercontent.com/6440455/147130690-3121a334-f2f3-453b-84ac-1579b3867296.png">

## Description

- Adds UI and functionality to the database backups tab
- Adds a mock for `/databases/:engine/instances/:databaseId/backups`

## Todo
- [x] Restore a backup functionality

## How to test

- Go to the backups tab for a Database (`/databases/3/backups` for example)
- Ensure you see 7 backups
- Check copy for spelling
- Ensure you can restore a backup (this is mocked but the flow works as if it was prod)
- Check the restore from backup confirmation modal for functionality, spelling, and style
